systemtests/src/main/java/io/enmasse/systemtest/iot/DefaultDeviceRegistry.java

```diff
@@ -262,27 +271,27 @@ private static ExternalJdbcDeviceConnectionServer externalH2ConnectionServer(fin
     }
 
     /*
-    public static DeviceRegistryServiceConfig newInfinispanBased() throws Exception {
-        var infinispanEndpoint = SystemtestsKubernetesApps.deployInfinispanServer();
-        return new DeviceRegistryServiceConfigBuilder()
-                .withNewInfinispan()
-                .withNewServer()
-
-                .withExternal(externalInfinispanRegistryServer(infinispanEndpoint))
-
-                .endServer()
-                .endInfinispan()
-                .build();
-    }
-
-    public static DeviceRegistryServiceConfig newFileBased() {
-        return new DeviceRegistryServiceConfigBuilder()
-                .withNewFile()
-                .withNumberOfDevicesPerTenant(100_000)
-                .endFile()
-                .build();
-    }
-    */
+     * public static DeviceRegistryServiceConfig newInfinispanBased() throws Exception {
```

Comment:
Do we still need this?

Reply:
I don't think so. I will create an additional PR to get it removed.